src-gui/src/main/java/org/openkilda/model/FlowInfo.java

```diff
@@ -40,6 +40,9 @@ public class FlowInfo implements Serializable {
     @JsonProperty("flowid")
     private String flowid;
 
+    @JsonProperty("y_flow_id")
+    private String yFlowId;
+
```

Comment:
I think this class can be simplified with the @JsonNaming(value = SnakeCaseStrategy.class) annotation, can't it?

Reply:
You are right, but for our case, this is not the best approach. In this class, we have snake case for some fields, kebab case for other fields. Moreover, we have mixes of two cases in one field. How about this:
    @JsonProperty("target-path-computation_strategy")
    private String targetPathComputationStrategy;
    
    
Although it is a good idea to use this class-level annotation @JsonNaming(value = SnakeCaseStrategy.class), I would leave it as it is for now and create an additional task to fix all these kinds of things.

Because you understand, if we want to unify everything in this class, then I have to prepare for this unification in the UI part, which could cause many more additional changes.

Reply:
An intermediate option could be to set the SnakeCaseStrategy and let the special cases be manually specified. Right now is not necessary to unification all the UI code but I agree we can do it later.

Reply:
fixed